gulpfile.mjs

```diff
@@ -42,6 +42,7 @@ const packagedir = path.resolve('./package');
 const feedPAT = argv.feedPAT || process.env['AZ_DevOps_Read_PAT'];
 const isOfficialBuild = argv.isOfficialBuild && argv.isOfficialBuild.toLowerCase() == "true";
 const isPreviewBuild = argv.isPreviewBuild && argv.isPreviewBuild.toLowerCase() == "true";
+const feedName = argv.useFeed || 'nuget.org';
```

Comment:
can we check once if nuget is the right default in case useFeed param is missing? or do we want to take the 2 other alternate present in gulpfile based on the right params

Reply:
Nuget.org was the previous default in place. The useFeed now acts like an override.

Reply:
![image](https://github.com/user-attachments/assets/5102434b-d572-4093-9679-83d0d4c05853)
We had explicitly set 'feedName' to 'nuget.org' as default.